tests/integration-tests/configs/isolated_regions.yaml

```diff
@@ -12,6 +12,11 @@ test-suites:
           instances: {{ INSTANCES }}
           oss: {{ OSS }}
           schedulers: {{ SCHEDULERS }}
+          benchmarks:
```

Comment:
Out of scope for this PR, but we must address this: benchmark fixture should be made optional